doc/devel/coding_guide.rst

```diff
@@ -145,7 +145,7 @@ Merging
   the PR.
 
   Ensure that all API changes are documented in the relevant file in
-  :file:`doc/api/next_api_changes` and significant new features have and
+  :file:`doc/api/recent_api_changes_3.3` and significant new features have and
```

Comment:
IMHO this doesn't make sense. I think there was a miscommunication. The folder should be named `api_changes_3.3`, so that it does not have to be renamed later.

And then all documentation references should use "the most recent `api_changes_X.Y`", e.g.
```suggestion
  the most recent :file:`doc/api/api_changes_X.Y` and significant new features have and
```

Reply:
Ok Got it. Thanks for the clarification.